<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiColorSettings</class>
 <widget class="QDialog" name="UiColorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="random_ch">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>Случайный</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Число генераций:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="generation_number">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>22</height>
           </size>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="random_value">
          <property name="text">
           <string>Корректировая яркость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="random_saturation">
          <property name="text">
           <string>Заменять насыщенность</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QRadioButton" name="enumer_ch">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>Перебор</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="colors_list">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="tb_rem">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>➖</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/image/icon/resourses/CMinus.ico</normaloff>:/image/icon/resourses/CMinus.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tb_add">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;; color: rgb(0, 222, 0)</string>
          </property>
          <property name="text">
           <string>➕</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/image/icon/resourses/CPlus.ico</normaloff>:/image/icon/resourses/CPlus.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
